graalwasm/graalwasm-micronaut-excelice/src/main/java/com/example/Controller.java

```diff
@@ -0,0 +1,104 @@
+package com.example;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Part;
+import io.micronaut.http.annotation.Post;
+import io.micronaut.http.multipart.CompletedFileUpload;
+import jakarta.inject.Inject;
+import io.micronaut.views.View;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+@io.micronaut.http.annotation.Controller
+public class Controller {
+
+    @Inject
+    private Repository repository;
+    private ExcelizeService service;
+
+    public Controller(ExcelizeService service){
+        this.service=service;
```

Comment:
Please run a code formatter on all Java files. There is a lot of redundant newlines and style errors like this:
```suggestion
        this.service = service;
```